lib/ex_machina.ex

```diff
@@ -49,6 +49,33 @@ defmodule ExMachina do
       def build_list(number_of_factories, factory_name, attrs \\ %{}) do
         ExMachina.build_list(__MODULE__, number_of_factories, factory_name, attrs)
       end
+
+      def create(_) do
+        raise_error_helper("create/1", "insert/1")
+      end
+
+      def create(_, _) do
+        raise_error_helper("create/2", "insert/2")
+      end
+
+      def create_pair(_, _) do
+        raise_error_helper("create_pair/2", "insert_pair/2")
+      end
+
+      def create_list(_, _, _) do
+        raise_error_helper("create_list/3", "insert_list/3")
+      end
+
+      defp raise_error_helper(old_function, new_function) do
+        raise """
+        #{old_function} has been removed
+        If you are using ExMachina.Ecto, use #{new_function} instead
+        If you are using ExMachina with a custom save_record/2, you now must use ExMachina.Strategy
+        See the ExMachina.Strategy documentation for more examples
```

Comment:
What do you think of removing "more" from this since there are no examples shown in the error message